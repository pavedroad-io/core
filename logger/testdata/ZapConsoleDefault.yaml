logpackage: zap
loglevel: info
enabletimestamps: false
enablecolorlevels: true
enablecloudevents: true
cloudeventscfg:
  setid: hmac
  hmackey: pavedroad
  source: http://github.com/pavedroad-io/core/go/logger
  specversion: "1.0"
  type: io.pavedroad.cloudevents.log
  setsubjectlevel: true
enablekafka: false
kafkaformat: cloudevents
kafkaproducercfg:
  brokers:
  - localhost:9092
  topic: logs
  partition: random
  key: fixed
  keyname: user
  compression: snappy
  ackwait: local
  prodflushfreq: 500ms
  prodretrymax: 10
  prodretryfreq: 100ms
  metaretrymax: 10
  metaretryfreq: 2s
  enabletls: false
  tlscfg: null
  enabledebug: false
enableconsole: true
consoleformat: text
consolewriter: ""
enablefile: false
fileformat: json
filelocation: testdata/ZapConsoleDefault.log
enablerotation: false
rotationcfg:
  maxsize: 0
  maxage: 0
  maxbackups: 0
  localtime: false
  compress: false
enabledebug: false
