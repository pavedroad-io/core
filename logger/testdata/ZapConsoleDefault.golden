[34mINFO[0m	Infof using zap	{"source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","type":"io.pavedroad.cloudevents.log"}
[33mWARN[0m	Warnf using zap	{"source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","type":"io.pavedroad.cloudevents.log"}
[31mERROR[0m	Errorf using zap	{"source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","type":"io.pavedroad.cloudevents.log"}
[34mINFO[0m	Print usingzap	{"source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","type":"io.pavedroad.cloudevents.log"}
[34mINFO[0m	Printf using zap	{"source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","type":"io.pavedroad.cloudevents.log"}
[34mINFO[0m	Println using zap	{"source":"http://github.com/pavedroad-io/core/go/logger","specversion":"1.0","type":"io.pavedroad.cloudevents.log"}
